templates/modals/delete_doc_confirm.html

```diff
@@ -1,17 +1,18 @@
 <div role="document" id="delete-confirm">
   <div class="modal-header">
     <button type="button" class="cancel close" ng-click="cancel()">&times;</button>
-    <h3>{{'Confirm' | translate}}</h3>
+    <h3 translate>Confirm</h3>
   </div>
   <div class="modal-body">
-    <p>{{'confirm.delete' | translate}}</p>
+    <p ng-show="model.docs.length === 1" translate>confirm.delete</p>
+    <p ng-show="model.docs.length !== 1" translate translate-values="{ number: {{model.docs.length}} }">confirm.delete.plural</p>
```

Comment:
Yeah we could just pass the length rather than the array. I don't think it'll make any difference to the JS engine. Stylistically I'm happy with either.